### ISSUE TITLE
ERROR brew upgrade 

### ISSUE THREAD

**Issue by shuxiao9058** [d12frosted/homebrew-emacs-plus#101]
```
➜  workspace brew upgrade -y
Updating Homebrew...
Error: Mojave supports only building from --HEAD
```

**Comment by d12frosted** [comment]
Well, as the errors says, you have to install using `--HEAD` option.

**Comment by crazymousethief** [comment]
Just do this
```shell
brew uninstall emacs-plus && brew install emacs-plus --HEAD
```
Related issue #95

**Comment by d12frosted** [comment]
Ah, the error is brought even if you have installed it with `--HEAD` option.

**Comment by d12frosted** [comment]
Commit 8216ee8d1942c6d2978ae9faabf050725da91333 should do the trick.

**Comment by d12frosted** [comment]
@shuxiao9058 thank you for reporting 😸

**Comment by d12frosted** [comment]
Ok, sorry. Today is not my day :( 

Here is the fix for regression eb2afa8

**Comment by d12frosted** [comment]
If there are no other complaints regarding this issue, I am clothing it.

Thanks everyone and sorry for the mess.
